lib/msal-angular/docs/v1-docs/cors-api.md

```diff
@@ -0,0 +1,24 @@
+# CORS API usage in MSAL Angular v2
```

Comment:
Should this be in v2 docs?

Reply:
Oh I see this is also in v2. Is this also applicable to v1?

Reply:
Ah. It is applicable (the sample code below is different for v1 and v2), but I need to change the title. Thanks for catching that!

Reply:
Changed.